### PR TITLE
Toggling 1 detection

### PR DESCRIPTION
**Pull request by unknownhad**
Toggling 1 detection

### CODE REVIEW

File: content/waf/change-log/2022-10-17.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  </thead>
+  <tbody>
+    <tr>
+      <td>Cloudflare Specials</td>
+      <td>...1bc977d1</td>
+      <td>100005</td>
+      <td>DotNetNuke - File Inclusion - CVE:CVE-2018-9126, CVE:CVE-2011-1892, CVE:CVE-2022-31474</td>
+      <td>N/A</td>
+      <td>Block</td>
+      <td>This detection was announces as ...845e3ec7 on new WAF</td>
```
Comment (pedrosousa) [inline content/waf/change-log/2022-10-17.md]:
```suggestion
      <td>This detection was announced as ...845e3ec7 on new WAF</td>
```

File: content/waf/change-log/2022-10-17.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+      <td>US Passport</td>
+      <td>Log</td>
+      <td>Disable</td>
+      <td>This detection is part of Sensitive Data Disclosure (SDD)</td>
+    </tr>
+    <tr>
+      <td>Sensitive Data Disclosure (SDD)</td>
+      <td>...0a290153</td>
+      <td>N/A</td>
+      <td>Wisconsin Driver's Licence</td>
```
Comment (pedrosousa) [inline content/waf/change-log/2022-10-17.md]:
```suggestion
      <td>Wisconsin Driver's License</td>
```
Let's use the same spelling.

Comment (unknownhad) [reply content/waf/change-log/2022-10-17.md]: Right, I missed this one.

File: content/waf/change-log/2022-10-17.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+      <td>Illinois Driver's License</td>
+      <td>Log</td>
+      <td>Disable</td>
+      <td>This detection is part of Sensitive Data Disclosure (SDD)</td>
+    </tr>
+    <tr>
+      <td>Sensitive Data Disclosure (SDD)</td>
+      <td>...9f7200b4</td>
+      <td>N/A</td>
+      <td>New York Driver's license</td>
```
Comment (pedrosousa) [inline content/waf/change-log/2022-10-17.md]:
```suggestion
      <td>New York Driver's License</td>
```
Also the same capitalization.

**Review by pedrosousa — APPROVED** [review]

**Pull request merged**